bin/start_sync_stop.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+"""A script to test rippled in an infinite loop of start-sync-stop.
+
+- Expects 'rippled' to be on the `PATH` and 'rippled.cfg' to be in the current
+  directory.
+- Takes no arguments.
+- Can be stopped with SIGINT.
+- Tested on Python 3.8.
+- Has no dependencies outside the standard library.
+"""
+
+import asyncio
+import contextlib
+import json
+import logging
+import os
+import subprocess
+import time
+import urllib.error
+import urllib.request
+
+
+@contextlib.asynccontextmanager
+async def rippled(path='rippled', config='rippled.cfg'):
+    """A context manager for a rippled process."""
+    # Start the server.
+    process = await asyncio.create_subprocess_exec(
+        path,
+        '--conf',
+        config,
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+    )
+    logging.info(f'rippled started with pid {process.pid}')
+    try:
+        yield process
+    finally:
+        # Ask it to stop.
+        logging.info(f'asking rippled ({process.pid}) to stop')
+        start = time.time()
+        process.terminate()
+
+        # Wait nicely.
+        try:
+            await asyncio.wait_for(process.wait(), 60)
```

Comment:
Running a debug version of the server locally, I find that a polite shutdown can take on the order of 90 seconds.  I suggest you crank the `await` timeout to `120`.